### ISSUE TITLE
Generating the Hebrew js file

### ISSUE THREAD

**Issue by scruffian** [openscriptures/strongs#5]
There is a perl script which generates the Greek js file, but I don't see a similar thing for the Hebrew one. Does anyone have a copy of it?

**Comment by jag3773** [comment]
I don't have it, maybe it was never written?

**Comment by scruffian** [comment]
There is a Hebrew js file so there must have been one at some point.

**Comment by DarrellHSmith** [comment]
I don't believe it ever was. NT Greek is far easier to grok and more broadly germane (I use Robinson-Pierpont). David Troidl's work on point for OT Hebrew has been greatly appreciated. ! 
 

    On Thursday, September 13, 2018 5:31 AM, Jesse Griffin <notifications@github.com> wrote:
 

 I don't have it, maybe it was never written?—

**Comment by DavidTroidl** [comment]
There is a StrongJson.xslt, which I would assume would do the job.  I haven't looked at this in years.  StrongHebrewG has been superseded by the [Hebrew Lexicon](https://github.com/openscriptures/HebrewLexicon).  See the readme for details, along with HebrewLexicon.pdf.
